tests/iuse_test.cpp

```diff
@@ -72,6 +73,14 @@ TEST_CASE( "eyedrops", "[iuse][eyedrops]" )
                 AND_THEN( "it removes the boomered effect" ) {
                     CHECK_FALSE( dummy.has_effect( effect_boomered ) );
                 }
+
+                REQUIRE( dummy.has_effect( effect_conjunctivitis ) );
```

Comment:
Looks like this is only going to sometimes happen? Need a mechanism to force it to happen, possibly by just adding another scenario that applies conjunctivitis explicitly.

Reply:
Oh yeah, nondeterministic tests *probably* aren't a good idea, good catch.